### DISCUSSION TITLE
Install Language

### DISCUSSION THREAD

**Discussion by DACN** [audacity/audacity#4658] [Ideas & Feature Requests]
I installed Audacity using choco - so I didn't get a choice of language. Not sure of the general behaviour, but either
(1) because I didn't get a choice, it defaulted to the first language alphabetically - Afrikaans
(2) maybe it tried my Windows language (Scots Gaelic !!), didn't find it, and reverted to the first language alphabetically

whatever - would suggest that something like English would be more generally understood.

All credit to your manual which helped in finding English as a choice by telling me to pick menu item second left etc.

**Comment by petersampsonaudacity** [comment]
Audacity uses "Sysrem" as the default for language setting - so what you get depends on your computer.
